scripts/src/primazatest/options/primaza-alice.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: primaza.io/v1alpha1
+kind: Tenant
+kubeconfig: ~/.kube/config
+manifestDirectory: ./out/config
+name: primaza-alice
+version: latest
+context: kind-primazactl-tenant-test
+internalUrl:
+clusterEnvironments:
+- name: worker-alice
+  environment: test
+  targetCluster:
+    context: kind-primazactl-join-test
+    internalUrl:
+    kubeconfig: ~/.kube/config
+  applicationNamespaces:
+  - name: alice-app
+  serviceNamespaces:
+  - name: alice-svc
```

Comment:
happy to have a control plane section but then `name` does't make sense because it is not the control plane name so 
maybe use  `namespace`. Also version applies to clusterEnvironments so better at the top level. Should we then also add control plane section under `clusterEnviornments`?

i went with manifest directory to keep it simple and we don't really want the user wiring their own and using different names. After I do the release work story the names will match those used in the primaza release. Also would really hope users use `version` instead.

Reply:
* on `name`: I agree. I'd suggest to borrow the approach from kubernetes objects and have a `metadata` field at the root level with a `name` property.
* on `version`: IIRC, we decided that initially we assume the agents are the same version of the ControlPlane. So for now we can skip having this data in `clusterEnvironment`. Depending on how we implement this, it may be needed to have this information in each `applicationNamespace` and `serviceNamespace`.
* on `manifestDir`: it's fine for me to refine the approach in a subsequent PR

Reply:
This is not making any sense to me. Agree we assume all are using the same version, mixing versions would be a testing nightmare and we don't even have 1 version yet.
Going off using control plane as we do nothing proactive to or with the control plane, we create a namespace and add a bunch of objects to it. Also I don't think we want to go the metadata route - it just over complicates stuff for no benefit. Personally I like the current structure and propose we leave as-is for now until we find a burning reason to change it.

Reply:
Have changed it to this:

```
apiVersion: primaza.io/v1alpha1
kind: Tenant
name: primaza-alice
manifestDirectory: ./out/config
version: latest
controlPlane:
    context: kind-primazactl-tenant-test
    kubeconfig: ~/.kube/config
    internalUrl:
clusterEnvironments:
- name: worker-alice
  environment: test
  targetCluster:
    context: kind-primazactl-join-test
    internalUrl:
    kubeconfig: ~/.kube/config
  applicationNamespaces:
  - name: alice-app
  serviceNamespaces:
  - name: alice-svc
```